src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/storage/S3StorageService.kt

```diff
@@ -18,30 +18,18 @@ class S3StorageService(
 
   private val log = LoggerFactory.getLogger(this::class.java)
 
-  override fun downloadFile(fileKey: UUID): ByteArray {
-    return s3Client.getObject(
-      GetObjectRequest.builder().bucket(bucketName).key(fileKey.toString()).build()
-    ).readAllBytes()
+  override fun downloadFile(fileS3Key: UUID): ByteArray {
+    log.debug("downloading file to s3://$bucketName/$fileS3Key")
+    val request = GetObjectRequest.builder().bucket(bucketName).key(fileS3Key.toString()).build()
+    return s3Client.getObject(request).readAllBytes()
   }
 
-  override fun uploadFile(fileBytes: ByteArray): S3BulkResponseEntity {
-    log.debug("Uploading file to s3://$bucketName")
-    val result: S3BulkResponseEntity
-    val uuid = UUID.randomUUID()
-    s3Client.putObject(
-      PutObjectRequest.builder().bucket(bucketName).key(uuid.toString()).build(),
-      RequestBody.fromBytes(fileBytes)
-    )
-      .sdkHttpResponse()
-      .also { log.info("uploaded $uuid to $bucketName: ${it.statusCode()}") }
-      .let { response ->
-        result = S3BulkResponseEntity(
-          bucket = bucketName,
-          fileKey = uuid,
-          successful = response.isSuccessful,
-          statusCode = response.statusCode()
-        )
-      }
-    return result
```

Comment:
Do we lose any error handling here?

Reply:
That code wasn't used at all. We can add it at a later stage if required.